k8s/config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: vineyardcloudnative/vineyard-operator
+  newName: localhost:5001/vineyard-operator
```

Comment:
?

Reply:
Done, what do you think about moving it to gitignore? As we run the e2e test locally, it will be changed if the e2e test fail.